src/NewsService/Startup.cs

```diff
@@ -76,8 +76,8 @@ public Startup(IConfiguration configuration)
                 .GetSection(BaseServiceInfoConfig.SectionName)
                 .Get<BaseServiceInfoConfig>();
 
-            Version = "1.2.4";
-            Description = "NewsService, is intended to work with the news - create them, update info and etc.";
+            Version = "1.2.5.2";
+            Description = "Rework news service";
```

Comment:
a, return message, it is not history log, it is api info

Reply:
done